packages/tooltips/src/styled/StyledParagraph.ts

```diff
@@ -16,8 +15,13 @@ const COMPONENT_ID = 'tooltip.paragraph';
  */
 export const StyledParagraph = styled.div.attrs({
```

Comment:
Consumers often put several paragraphs of content within the this boundary. Especially with it being ready by an `aria-describedby` usage I don't think this would be necessary.